src/app/containers/webVitals/index.jsx

```diff
@@ -0,0 +1,29 @@
+// Hooks
```

Comment:
**nitpick**: Could we use camel case for the folder name - `WebVitals` instead of `webVitals`

Reply:
Good spot, Updated 👍